scss/angular/overrides/_text-angular.scss

```diff
@@ -0,0 +1,54 @@
+$crm-text-angular-min-height: 120px;
```

Comment:
put this inside the `[text-angular]` scope, given you're not using it anywhere else